torch/csrc/jit/codegen/cuda/lower2device.cpp

```diff
@@ -347,7 +349,7 @@ void GpuLower::lower(Fusion* fusion, DataType index_type) {
 
   // Reorder expressions for loop-nest generation respecting computeAt
   // relationships
-  const auto exprs_sorted = reorderExprsForComputeAt();
+  auto exprs_sorted = reorderExprsForComputeAt();
```

Comment:
It can, but may not be as is. That's what I meant to say in the above:

> It is also possible to reuse the sorted expression list but not always as we also replace some expressions in the Fusion container at the beginning of lowering, so the sorted list needs to be also updated. It doesn't mean we would need to do the complete sorting again but we could just update the list as necessary. I didn't try that, but I'd revisit if sorting is really costly.